src/plugins/workspace/public/plugin.ts

```diff
@@ -374,6 +375,18 @@ export class WorkspacePlugin
       workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
     });
 
+    // register workspace navigation
+    core.application.register({
+      id: WORKSPACE_NAVIGATION_APP_ID,
+      title: '',
+      navLinkStatus: AppNavLinkStatus.hidden,
+      async mount(params: AppMountParameters) {
+        const { renderNavigationApp } = await import('./application');
```

Comment:
Actually we can just navigate to the desired app without rendering a page. That should speed up the redirect.

Reply:
Sure, the code has been updated